src/subscription_manager/entcertlib.py

```diff
@@ -465,7 +429,7 @@ def updates(self) -> int:
         return len(self.added) + len(self.rogue)
 
     # need an ExceptionsReport?
-    # FIXME: needs to be properties
+    @property
```

Comment:
note that this change breaks the `AttachCommand`:
https://github.com/candlepin/subscription-manager/blob/b5c54bfc2e0202e8a2e069cdb0b4b845a1ce0a71/src/subscription_manager/cli_command/attach.py#L279-L281

I think this property change would be done better in its own commit.